docs/operations/cpu_hotplug.md

```diff
@@ -0,0 +1,134 @@
+# CPU Hotplug
+
+Starting kubevirt 1.0 its possible to configure the VM workload
+to allow for adding or removing the virtual CPUs while the VM is running.
+
+### Abstract
+A **virtual CPU** is the CPU that is seen to the Guest VM OS. A VM owner can manage the amount of vCPUs from the VM spec template using the CPU topology fields (`spec.template.spec.domain.cpu`). The `cpu` object has the integers `cores,sockets,threads` so that the virtual CPU is calculated by the following formula: `cores * sockets * threads`. Before CPU hotplug was introduced the VM owner could change these integers in the VM template while the VM is running, and they were staged until the next boot cycle. With CPU hotplug it is possible to patch the `sockets` integer in the VM template and the change will take effect right away. The amount of vCPUs can be changed from 1 vCPU up to the limit of `cores * threads * maxSockets`. The `maxSockets` integer is pre-calculated by kubevirt but can also be managed by the VM owner or by the cluster administrator.
```

Comment:
```suggestion
A **virtual CPU** (vCPU) is the CPU that is seen to the Guest VM OS. A VM owner can manage the amount of vCPUs from the VM spec template using the CPU topology fields (`spec.template.spec.domain.cpu`). The `cpu` object has the integers `cores,sockets,threads` so that the virtual CPU is calculated by the following formula: `cores * sockets * threads`. 

Before CPU hotplug was introduced, the VM owner could change these integers in the VM template while the VM is running, and they were staged until the next boot cycle. With CPU hotplug, it is possible to patch the `sockets` integer in the VM template and the change will take effect right away. 

The amount of vCPUs can be changed from 1 vCPU up to the limit of `cores * threads * maxSockets`. The `maxSockets` integer is pre-calculated by KubeVirt but can also be managed by the VM owner or by the cluster administrator.
```